packages/cozy-jobs-cli/src/cozy-authenticate.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 
+/* eslint no-console: off */
```

Comment:
I think we must use the logger

Or put this rule in the global config file

Reply:
In this case, console.log is used to display a message to the user of the cozy-run-dev cli. I think this is justified.